

<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1038/s41587-022-01566-x" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1038/s41587-022-01566-x" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">July 2023</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>High-throughput, targeted MHC class I immunopeptidomics using a functional genetics screening platform</h3>
            Identification of CD8+ T cell epitopes is critical for the development of immunotherapeutics. Existing methods for major histocompatibility complex class I (MHC class I) ligand discovery are time intensive, specialized and unable to interrogate specific proteins on a large scale. Here, we present EpiScan, which uses surface MHC class I levels as a readout for whether a genetically encoded peptide is an MHC class I ligand. Predeterm
            <span class="dots">...</span>
            <span class="more">
            ined starting pools composed of >100,000 peptides can be designed using oligonucleotide synthesis, permitting large-scale MHC class I screening. We exploit this programmability of EpiScan to uncover an unappreciated role for cysteine that increases the number of predicted ligands by 9-21%, reveal affinity hierarchies by analysis of biased anchor peptide libraries and screen viral proteomes for MHC class I ligands. Using these data, we generate and iteratively refine peptide binding predictions to create EpiScan Predictor. EpiScan Predictor performs comparably to other state-of-the-art MHC class I peptide binding prediction algorithms without suffering from underrepresentation of cysteine-containing peptides. Thus, targeted immunopeptidomics using EpiScan will accelerate CD8+ T cell epitope discovery toward the goal of individual-specific immunotherapeutics.
            </br><a style="float: right;" href="https://doi.org/10.1038/s41587-022-01566-x">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1038/s41594-023-01023-6" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1038/s41594-023-01023-6" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">July 2023</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Structural and functional characteristics of SARS-CoV-2 Omicron subvariant BA.2 spike</h3>
            The Omicron subvariant BA.2 has become the dominant circulating strain of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in many countries. Here, we have characterized structural, functional and antigenic properties of the full-length BA.2 spike (S) protein and compared replication of the authentic virus in cell culture and an animal model with previously prevalent variants. BA.2 S can fuse membranes slightly more eff
            <span class="dots">...</span>
            <span class="more">
            iciently than Omicron BA.1, but still less efficiently than other previous variants. Both BA.1 and BA.2 viruses replicated substantially faster in animal lungs than the early G614 (B.1) strain in the absence of pre-existing immunity, possibly explaining the increased transmissibility despite their functionally compromised spikes. As in BA.1, mutations in the BA.2 S remodel its antigenic surfaces, leading to strong resistance to neutralizing antibodies. These results suggest that both immune evasion and replicative advantage may contribute to the heightened transmissibility of the Omicron subvariants.
            </br><a style="float: right;" href="https://doi.org/10.1038/s41594-023-01023-6">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.15252/embr.202357724" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.15252/embr.202357724" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">December 2023</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Antibody-mediated SARS-CoV-2 entry in cultured cells</h3>
            Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) enters host cells by first engaging its cellular receptor angiotensin converting enzyme 2 (ACE2) to induce conformational changes in the virus-encoded spike protein and fusion between the viral and target cell membranes. Here, we report that certain monoclonal neutralizing antibodies against distinct epitopic regions of the receptor-binding domain of the spike can replace
            <span class="dots">...</span>
            <span class="more">
             ACE2 to serve as a receptor and efficiently support membrane fusion and viral infectivity in vitro. These receptor-like antibodies can function in the form of a complex of their soluble immunoglobulin G with Fc-gamma receptor I, a chimera of their antigen-binding fragment with the transmembrane domain of ACE2 or a membrane-bound B cell receptor, indicating that ACE2 and its specific interaction with the spike protein are dispensable for SARS-CoV-2 entry. These results suggest that antibody responses against SARS-CoV-2 may help expand the viral tropism to otherwise nonpermissive cell types with potential implications for viral transmission and pathogenesis.
            </br><a style="float: right;" href="https://doi.org/10.15252/embr.202357724">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1182/blood.2022018903" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1182/blood.2022018903" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">August 2023</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Blood group A enhances SARS-CoV-2 infection</h3>
            Among the risk factors for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), ABO(H) blood group antigens are among the most recognized predictors of infection. However, the mechanisms by which ABO(H) antigens influence susceptibility to COVID-19 remain incompletely understood. The receptor-binding domain (RBD) of SARS-CoV-2, which facilitates host cell engagement, bears significant similarity to galectins, an ancient fa
            <span class="dots">...</span>
            <span class="more">
            mily of carbohydrate-binding proteins. Because ABO(H) blood group antigens are carbohydrates, we compared the glycan-binding specificity of SARS-CoV-2 RBD with that of galectins. Similar to the binding profile of several galectins, the RBDs of SARS-CoV-2, including Delta and Omicron variants, exhibited specificity for blood group A. Not only did each RBD recognize blood group A in a glycan array format, but each SARS-CoV-2 virus also displayed a preferential ability to infect blood group A-expressing cells. Preincubation of blood group A cells with a blood group-binding galectin specifically inhibited the blood group A enhancement of SARS-CoV-2 infection, whereas similar incubation with a galectin that does not recognize blood group antigens failed to impact SARS-CoV-2 infection. These results demonstrated that SARS-CoV-2 can engage blood group A, providing a direct link between ABO(H) blood group expression and SARS-CoV-2 infection.
            </br><a style="float: right;" href="https://doi.org/10.1182/blood.2022018903">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1126/science.adc9498" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1126/science.adc9498" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">April 2023</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Germline-encoded amino acid-binding motifs drive immunodominant public antibody responses</h3>
            Despite the vast diversity of the antibody repertoire, infected individuals often mount antibody responses to precisely the same epitopes within antigens. The immunological mechanisms underpinning this phenomenon remain unknown. By mapping 376 immunodominant "public epitopes" at high resolution and characterizing several of their cognate antibodies, we concluded that germline-encoded sequences in antibodies drive recurrent recognit
            <span class="dots">...</span>
            <span class="more">
            ion. Systematic analysis of antibody-antigen structures uncovered 18 human and 21 partially overlapping mouse germline-encoded amino acid-binding (GRAB) motifs within heavy and light V gene segments that in case studies proved critical for public epitope recognition. GRAB motifs represent a fundamental component of the immune system's architecture that promotes recognition of pathogens and leads to species-specific public antibody responses that can exert selective pressure on pathogens.
            </br><a style="float: right;" href="https://doi.org/10.1126/science.adc9498">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1016/j.semarthrit.2023.152177" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1016/j.semarthrit.2023.152177" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">April 2023</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Humoral and cellular immune responses in persons with rheumatoid arthritis after a third dose of mRNA COVID-19 vaccine</h3>
            Objective: Disease-modifying anti-rheumatic drugs (DMARDs) that treat rheumatoid arthritis (RA) may reduce immune responses to COVID-19 vaccination. We compared humoral and cell-mediated immunity before and after a 3rd dose of mRNA COVID vaccine in RA subjects. Methods: RA patients that received 2 doses of mRNA vaccine enrolled in an observational study in 2021 before receiving a 3rd dose. Subjects self-reported holding or continui
            <span class="dots">...</span>
            <span class="more">
            ng DMARDs. Blood samples were collected pre- and 4 weeks after the 3rd dose. 50 healthy controls provided blood samples. Humoral response was measured with in-house ELISA assays for anti-Spike IgG (anti-S) and anti-receptor binding domain IgG (anti-RBD). T cell activation was measured after stimulation with SARS-CoV-2 peptide. Spearman's correlations assessed the relationship between anti-S, anti-RBD, and frequencies of activated T cells. Results: Among 60 subjects, mean age was 63 years and 88% were female. 57% of subjects held at least 1 DMARD around the 3rd dose. 43% (anti-S) and 62% (anti-RBD) had a normal humoral response at week 4, defined as ELISA within 1 standard deviation of the healthy control mean. No differences in antibody levels were observed based on holding DMARDs. Median frequency of activated CD4 T cells was significantly greater post- vs. pre-3rd dose. Changes in antibody levels did not correlate with change in frequency of activated CD4 T cells. Conclusion: Virus-specific IgG levels significantly increased in RA subjects using DMARDs after completing the primary vaccine series, though fewer than two-thirds achieved a humoral response like healthy controls. Humoral and cellular changes were not correlated.
            </br><a style="float: right;" href="https://doi.org/10.1016/j.semarthrit.2023.152177">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1126/sciimmunol.abl9464" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1126/sciimmunol.abl9464" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">July 2022</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>SARS-CoV-2 epitope-specific CD4+ memory T cell responses across COVID-19 disease severity and antibody durability</h3>
            CD4+ T cells are central to long-term immunity against viruses through the functions of T helper 1 (TH1) and T follicular helper (TFH) cell subsets. To better understand the role of these subsets in coronavirus disease 2019 (COVID-19) immunity, we conducted a longitudinal study of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)-specific CD4+ T cell and antibody responses in convalescent individuals who seroconverted du
            <span class="dots">...</span>
            <span class="more">
            ring the first wave of the pandemic in Boston, MA, USA, across a range of COVID-19 disease severities. Analyses of spike (S) and nucleocapsid (N) epitope-specific CD4+ T cells using peptide and major histocompatibility complex class II (pMHCII) tetramers demonstrated expanded populations of T cells recognizing the different SARS-CoV-2 epitopes in most individuals compared with prepandemic controls. Individuals who experienced a milder disease course not requiring hospitalization had a greater percentage of circulating TFH (cTFH) and TH1 cells among SARS-CoV-2-specific cells. Analysis of SARS-CoV-2-specific CD4+ T cells responses in a subset of individuals with sustained anti-S antibody responses after viral clearance also revealed an increased proportion of memory cTFH cells. Our findings indicate that efficient early disease control also predicts favorable long-term adaptive immunity.
            </br><a style="float: right;" href="https://doi.org/10.1126/sciimmunol.abl9464">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1038/s41588-022-01113-z" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1038/s41588-022-01113-z" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">August 2022</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Genetic regulation of OAS1 nonsense-mediated decay underlies association with risk of severe COVID-19</h3>
            The chr12q24.13 locus encoding OAS1-OAS3 antiviral proteins has been associated with coronavirus disease 2019 (COVID-19) susceptibility. Here, we report genetic, functional and clinical insights into this locus in relation to COVID-19 severity. In our analysis of patients of European (n = 2,249) and African (n = 835) ancestries with hospitalized versus nonhospitalized COVID-19, the risk of hospitalized disease was associated with a
            <span class="dots">...</span>
            <span class="more">
             common OAS1 haplotype, which was also associated with reduced severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) clearance in a clinical trial with pegIFN-λ1. Bioinformatic analyses and in vitro studies reveal the functional contribution of two associated OAS1 exonic variants comprising the risk haplotype. Derived human-specific alleles rs10774671-A and rs1131454 -A decrease OAS1 protein abundance through allele-specific regulation of splicing and nonsense-mediated decay (NMD). We conclude that decreased OAS1 expression due to a common haplotype contributes to COVID-19 severity. Our results provide insight into molecular mechanisms through which early treatment with interferons could accelerate SARS-CoV-2 clearance and mitigate against severe COVID-19.
            </br><a style="float: right;" href="https://doi.org/10.1038/s41588-022-01113-z">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1016/j.celrep.2022.110729" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1016/j.celrep.2022.110729" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">April 2022</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Structural and functional impact by SARS-CoV-2 Omicron spike mutations</h3>
            The Omicron variant of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), bearing an unusually high number of mutations, has become a dominant strain in many countries within several weeks. We report here structural, functional, and antigenic properties of its full-length spike (S) protein with a native sequence in comparison with those of previously prevalent variants. Omicron S requires a substantially higher level of 
            <span class="dots">...</span>
            <span class="more">
            host receptor ACE2 for efficient membrane fusion than other variants, possibly explaining its unexpected cellular tropism. Mutations not only remodel the antigenic structure of the N-terminal domain of the S protein but also alter the surface of the receptor-binding domain in a way not seen in other variants, consistent with its remarkable resistance to neutralizing antibodies. These results suggest that Omicron S has acquired an extraordinary ability to evade host immunity by excessive mutations, which also compromise its fusogenic capability.
            </br><a style="float: right;" href="https://doi.org/10.1016/j.celrep.2022.110729">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1172/jci.insight.143129" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1172/jci.insight.143129" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">January 2021</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Dissecting strategies to tune the therapeutic potential of SARS-CoV-2-specific monoclonal antibody CR3022</h3>
            The rapid spread of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), coupled with a lack of therapeutics, has paralyzed the globe. Although significant effort has been invested in identifying antibodies that block infection, the ability of antibodies to target infected cells through Fc interactions may be vital to eliminate the virus. To explore the role of Fc activity in SARS-CoV-2 immunity, the functional potential o
            <span class="dots">...</span>
            <span class="more">
            f a cross-SARS-reactive antibody, CR3022, was assessed. CR3022 was able to broadly drive antibody effector functions, providing critical immune clearance at entry and upon egress. Using selectively engineered Fc variants, no protection was observed after administration of WT IgG1 in mice or hamsters. Conversely, the functionally enhanced Fc variant resulted in increased pathology in both the mouse and hamster models, causing weight loss in mice and enhanced viral replication and weight loss in the more susceptible hamster model, highlighting the pathological functions of Fc-enhancing mutations. These data point to the critical need for strategic Fc engineering for the treatment of SARS-CoV-2 infection.
            </br><a style="float: right;" href="https://doi.org/10.1172/jci.insight.143129">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1126/science.abl9463" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1126/science.abl9463" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">December 2021</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Membrane fusion and immune evasion by the spike protein of SARS-CoV-2 Delta variant</h3>
            The Delta variant of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has outcompeted previously prevalent variants and become a dominant strain worldwide. We report the structure, function, and antigenicity of its full-length spike (S) trimer as well as those of the Gamma and Kappa variants, and compare their characteristics with the G614, Alpha, and Beta variants. Delta S can fuse membranes more efficiently at low lev
            <span class="dots">...</span>
            <span class="more">
            els of cellular receptor angiotensin converting enzyme 2 (ACE2), and its pseudotyped viruses infect target cells substantially faster than the other five variants, possibly accounting for its heightened transmissibility. Each variant shows different rearrangement of the antigenic surface of the amino-terminal domain of the S protein but only makes produces changes in the receptor binding domain (RBD), making the RBD a better target for therapeutic antibodies.
            </br><a style="float: right;" href="https://doi.org/10.1126/science.abl9463">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1126/science.abi9745" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1126/science.abi9745" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">August 2021</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Structural basis for enhanced infectivity and immune evasion of SARS-CoV-2 variants</h3>
            Several fast-spreading variants of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) have become the dominant circulating strains in the COVID-19 pandemic. We report here cryo-electron microscopy structures of the full-length spike (S) trimers of the B.1.1.7 and B.1.351 variants, as well as their biochemical and antigenic properties. Amino acid substitutions in the B.1.1.7 protein increase both the accessibility of its r
            <span class="dots">...</span>
            <span class="more">
            eceptor binding domain and the binding affinity for receptor angiotensin-converting enzyme 2 (ACE2). The enhanced receptor engagement may account for the increased transmissibility. The B.1.351 variant has evolved to reshape antigenic surfaces of the major neutralizing sites on the S protein, making it resistant to some potent neutralizing antibodies. These findings provide structural details on how SARS-CoV-2 has evolved to enhance viral fitness and immune evasion.
            </br><a style="float: right;" href="https://doi.org/10.1126/science.abi9745">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1016/j.jim.2020.112832" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1016/j.jim.2020.112832" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">October 2020</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>SARS-CoV-2-specific ELISA development</h3>
            Critical to managing the spread of COVID-19 is the ability to diagnose infection and define the acquired immune response across the population. While genomic tests for the novel Several Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) detect the presence of viral RNA for a limited time frame, when the virus is shed in the upper respiratory tract, tests able to define exposure and infection beyond this short window of detectabl
            <span class="dots">...</span>
            <span class="more">
            e viral replication are urgently needed. Following infection, antibodies are generated within days, providing a durable read-out and archive of exposure and infection. Several antibody tests have emerged to diagnose SARS-CoV-2. Here we report on a qualified quantitative ELISA assay that displays all the necessary characteristics for high-throughput sample analysis. Collectively, this test offers a quantitative opportunity to define both exposure and levels of immunity to SARS-CoV-2.
            </br><a style="float: right;" href="https://doi.org/10.1016/j.jim.2020.112832">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1126/science.abd4250" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1126/science.abd4250" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">November 2020</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Viral epitope profiling of COVID-19 patients reveals cross-reactivity and correlates of severity</h3>
            Understanding humoral responses to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is critical for improving diagnostics, therapeutics, and vaccines. Deep serological profiling of 232 coronavirus disease 2019 (COVID-19) patients and 190 pre-COVID-19 era controls using VirScan revealed more than 800 epitopes in the SARS-CoV-2 proteome, including 10 epitopes likely recognized by neutralizing antibodies. Preexisting antib
            <span class="dots">...</span>
            <span class="more">
            odies in controls recognized SARS-CoV-2 ORF1, whereas only COVID-19 patient antibodies primarily recognized spike protein and nucleoprotein. A machine learning model trained on VirScan data predicted SARS-CoV-2 exposure history with 99% sensitivity and 98% specificity; a rapid Luminex-based diagnostic was developed from the most discriminatory SARS-CoV-2 peptides. Individuals with more severe COVID-19 exhibited stronger and broader SARS-CoV-2 responses, weaker antibody responses to prior infections, and higher incidence of cytomegalovirus and herpes simplex virus 1, possibly influenced by demographic covariates. Among hospitalized patients, males produce stronger SARS-CoV-2 antibody responses than females.
            </br><a style="float: right;" href="https://doi.org/10.1126/science.abd4250">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1126/science.abc6284" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1126/science.abc6284" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">August 2020</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>DNA vaccine protection against SARS-CoV-2 in rhesus macaques</h3>
            The global coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has made the development of a vaccine a top biomedical priority. In this study, we developed a series of DNA vaccine candidates expressing different forms of the SARS-CoV-2 spike (S) protein and evaluated them in 35 rhesus macaques. Vaccinated animals developed humoral and cellular immune responses, includi
            <span class="dots">...</span>
            <span class="more">
            ng neutralizing antibody titers at levels comparable to those found in convalescent humans and macaques infected with SARS-CoV-2. After vaccination, all animals were challenged with SARS-CoV-2, and the vaccine encoding the full-length S protein resulted in >3.1 and >3.7 log10 reductions in median viral loads in bronchoalveolar lavage and nasal mucosa, respectively, as compared with viral loads in sham controls. Vaccine-elicited neutralizing antibody titers correlated with protective efficacy, suggesting an immune correlate of protection. These data demonstrate vaccine protection against SARS-CoV-2 in nonhuman primates.
            </br><a style="float: right;" href="https://doi.org/10.1126/science.abc6284">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1182/blood-2018-09-873125" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1182/blood-2018-09-873125" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">February 2019</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Plateletpheresis-associated lymphopenia in frequent platelet donors</h3>
            More than 1 million apheresis platelet collections are performed annually in the United States. After 2 healthy plateletpheresis donors were incidentally found to have low CD4+ T-lymphocyte counts, we investigated whether plateletpheresis causes lymphopenia. We conducted a cross-sectional single-center study of platelet donors undergoing plateletpheresis with the Trima Accel, which removes leukocytes continuously with its leukoredu
            <span class="dots">...</span>
            <span class="more">
            ction system chamber. We recruited 3 groups of platelet donors based on the total number of plateletpheresis sessions in the prior 365 days: 1 or 2, 3 to 19, or 20 to 24. CD4+ T-lymphocyte counts were <200 cells per microliter in 0/20, 2/20, and 6/20 donors, respectively (P = .019), and CD8+ T-lymphocyte counts were low in 0/20, 4/20, and 11/20 donors, respectively (P < .001). The leukoreduction system chamber's lymphocyte-extraction efficiency was ∼15% to 20% for all groups. Immunophenotyping showed decreases in naive CD4+ T-lymphocyte and T helper 17 (Th17) cell percentages, increases in CD4+ and CD8+ effector memory, Th1, and regulatory T cell percentages, and stable naive CD8+ and Th2 percentages across groups. T-cell receptor repertoire analyses showed similar clonal diversity in all groups. Donor screening questionnaires supported the good health of the donors, who tested negative at each donation for multiple pathogens, including HIV. Frequent plateletpheresis utilizing a leukoreduction system chamber is associated with CD4+ and CD8+ T-cell lymphopenia in healthy platelet donors. The mechanism may be repeated extraction of these cells during plateletpheresis. The cytopenias do not appear to be harmful.
            </br><a style="float: right;" href="https://doi.org/10.1182/blood-2018-09-873125">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1016/j.jaci.2017.11.019" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1016/j.jaci.2017.11.019" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">August 2018</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>The neonatal window of opportunity-early priming for life [editorial]</h3>
            The concept of the neonatal window of opportunity assigns the early postnatal period a critical role for lifelong host-microbial and immune homeostasis. It is supported by epidemiological evidence that links postnatal environmental exposure with disease susceptibility and mechanisms in the neonate host that facilitate the postnatal transposition, establish a stable microbiome, and promote immune maturation. During the conference on
            <span class="dots">...</span>
            <span class="more">
             “The neonatal window of opportunity – early priming for life,” postnatal micro-biome and immune maturation, epidemiological evidence, and fundamental mechanisms were discussed to identify new targets for future preventive and interventional measures.  From December 5 to 7, 2016, the Herrenhausen Conference “The neonatal window of opportunity – early priming for life” took place at Hannover, Germany, sponsored by the Volkswagen Foundation. The concept of the “neonatal window of opportunity,” that is, a critical nonredundant time frame in a newborn’s life during which environmental factors drive immune and tissue maturation and influence the susceptibility to immune-mediated and other diseases in adult life, was discussed.
            </br><a style="float: right;" href="https://doi.org/10.1016/j.jaci.2017.11.019">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1016/j.immuni.2016.12.013" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1016/j.immuni.2016.12.013" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">January 2017</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Lactobacillus-Deficient Cervicovaginal Bacterial Communities Are Associated with Increased HIV Acquisition in Young South African Women</h3>
            Elevated inflammation in the female genital tract is associated with increased HIV risk. Cervicovaginal bacteria modulate genital inflammation; however, their role in HIV susceptibility has not been elucidated. In a prospective cohort of young, healthy South African women, we found that individuals with diverse genital bacterial communities dominated by anaerobes other than Gardnerella were at over 4-fold higher risk of acquiring H
            <span class="dots">...</span>
            <span class="more">
            IV and had increased numbers of activated mucosal CD4+ T cells compared to those with Lactobacillus crispatus-dominant communities. We identified specific bacterial taxa linked with reduced (L. crispatus) or elevated (Prevotella, Sneathia, and other anaerobes) inflammation and HIV infection and found that high-risk bacteria increased numbers of activated genital CD4+ T cells in a murine model. Our results suggest that highly prevalent genital bacteria increase HIV risk by inducing mucosal HIV target cells. These findings might be leveraged to reduce HIV acquisition in women living in sub-Saharan Africa.
            </br><a style="float: right;" href="https://doi.org/10.1016/j.immuni.2016.12.013">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1016/j.jaip.2015.12.019" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1016/j.jaip.2015.12.019" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">May-June 2016</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Safety, Costs, and Efficacy of Rapid Drug Desensitizations to Chemotherapy and Monoclonal Antibodies</h3>
            Background: Rapid drug desensitization (RDD) is used to address hypersensitivity reactions to chemotherapeutics and monoclonal antibodies, allowing patients to be treated with optimal pharmacological agents. RDD protocols are tailored to each individual patient's reaction and needs, and protect against anaphylaxis, but overall risks, costs, and benefits have not been determined. Objective: We investigated the safety, efficacy, cost
            <span class="dots">...</span>
            <span class="more">
            s, and life expectancy of patients in a large population undergoing RDD. Methods: We analyzed 2177 RDD procedures performed in 370 patients with cancer, vasculitis, and hematological and connective tissue diseases who presented 402 reactions. A subgroup of carboplatin allergic patients with ovarian cancer treated with RDD was analyzed for costs and life expectancy and compared with a nonallergic control group. Results: RDD allowed all patients to receive safely the full dose of the medication to which they were reactive. A gradual increase in the fraction of outpatient desensitizations from 81% to 98% was achieved through risk stratification. Of the 2177 desensitizations, 93% had no or mild reactions whereas 7% had moderate to severe reactions, which did not preclude the completion of the treatment, and there were no deaths. Overall health costs in the carboplatin allergic group were not higher than those in the nonallergic group treated with standard of care. Administration of carboplatin through RDD was as effective as standard administration with a nonsignificant increase in life expectancy in desensitized patients as compared with nonallergic, nondesensitized controls. Conclusions: RDD is cost effective and safe for allergic patients with cancer and chronic disease to remain on first line therapy.
            </br><a style="float: right;" href="https://doi.org/10.1016/j.jaip.2015.12.019">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1073/pnas.1510824112" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1073/pnas.1510824112" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">August 2015</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Multitarget, quantitative nanoplasmonic electrical field-enhanced resonating device (NE2RD) for diagnostics</h3>
            Recent advances in biosensing technologies present great potential for medical diagnostics, thus improving clinical decisions. However, creating a label-free general sensing platform capable of detecting multiple biotargets in various clinical specimens over a wide dynamic range, without lengthy sample-processing steps, remains a considerable challenge. In practice, these barriers prevent broad applications in clinics and at patien
            <span class="dots">...</span>
            <span class="more">
            ts' homes. Here, we demonstrate the nanoplasmonic electrical field-enhanced resonating device (NE(2)RD), which addresses all these impediments on a single platform. The NE(2)RD employs an immunodetection assay to capture biotargets, and precisely measures spectral color changes by their wavelength and extinction intensity shifts in nanoparticles without prior sample labeling or preprocessing. We present through multiple examples, a label-free, quantitative, portable, multitarget platform by rapidly detecting various protein biomarkers, drugs, protein allergens, bacteria, eukaryotic cells, and distinct viruses. The linear dynamic range of NE(2)RD is five orders of magnitude broader than ELISA, with a sensitivity down to 400 fg/mL This range and sensitivity are achieved by self-assembling gold nanoparticles to generate hot spots on a 3D-oriented substrate for ultrasensitive measurements. We demonstrate that this precise platform handles multiple clinical samples such as whole blood, serum, and saliva without sample preprocessing under diverse conditions of temperature, pH, and ionic strength. The NE(2)RD's broad dynamic range, detection limit, and portability integrated with a disposable fluidic chip have broad applications, potentially enabling the transition toward precision medicine at the point-of-care or primary care settings and at patients' homes.
            </br><a style="float: right;" href="https://doi.org/10.1073/pnas.1510824112">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1016/j.cell.2013.05.023" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1016/j.cell.2013.05.023" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">June 2013</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>53BP1 mediates productive and mutagenic DNA repair through distinct phosphoprotein interactions</h3>
            The DNA damage response (DDR) protein 53BP1 protects DNA ends from excessive resection in G1, and thereby favors repair by nonhomologous end-joining (NHEJ) as opposed to homologous recombination (HR). During S phase, BRCA1 antagonizes 53BP1 to promote HR. The pro-NHEJ and antirecombinase functions of 53BP1 are mediated in part by RIF1, the only known factor that requires 53BP1 phosphorylation for its recruitment to double-strand br
            <span class="dots">...</span>
            <span class="more">
            eaks (DSBs). Here, we show that a 53BP1 phosphomutant, 53BP18A, comprising alanine substitutions of the eight most N-terminal S/TQ phosphorylation sites, mimics 53BP1 deficiency by restoring genome stability in BRCA1-deficient cells yet behaves like wild-type 53BP1 with respect to immunoglobulin class switch recombination (CSR). 53BP18A recruits RIF1 but fails to recruit the DDR protein PTIP to DSBs, and disruption of PTIP phenocopies 53BP18A. We conclude that 53BP1 promotes productive CSR and suppresses mutagenic DNA repair through distinct phosphodependent interactions with RIF1 and PTIP.
            </br><a style="float: right;" href="https://doi.org/10.1016/j.cell.2013.05.023">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1073/pnas.1121458109" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1073/pnas.1121458109" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">February 2012</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Functional redundancy between repair factor XLF and damage response mediator 53BP1 in V(D)J recombination and DNA repair</h3>
            The classical nonhomologous DNA end-joining (C-NHEJ) double-strand break (DSB) repair pathway in mammalian cells maintains genome stability and is required for V(D)J recombination and lymphocyte development. Mutations in the XLF C-NHEJ factor or ataxia telangiectasia-mutated (ATM) DSB response protein cause radiosensitivity and immunodeficiency in humans. Although potential roles for XLF in C-NHEJ are unknown, ATM activates a gener
            <span class="dots">...</span>
            <span class="more">
            al DSB response by phosphorylating substrates, including histone H2AX and 53BP1, which are assembled into chromatin complexes around DSBs. In mice, C-NHEJ, V(D)J recombination, and lymphocyte development are, at most, modestly impaired in the absence of XLF or ATM, but are severely impaired in the absence of both. Redundant functions of XLF and ATM depend on ATM kinase activity; correspondingly, combined XLF and H2AX deficiency severely impairs V(D)J recombination, even though H2AX deficiency alone has little impact on this process. These and other findings suggest that XLF may provide functions that overlap more broadly with assembled DSB response factors on chromatin. As one test of this notion, we generated mice and cells with a combined deficiency for XLF and 53BP1. In this context, 53BP1 deficiency, although leading to genome instability, has only modest effects on V(D)J recombination or lymphocyte development. Strikingly, we find that combined XLF/53BP1 deficiency in mice severely impairs C-NHEJ, V(D)J recombination, and lymphocyte development while also leading to general genomic instability and growth defects. We conclude that XLF is functionally redundant with multiple members of the ATM-dependent DNA damage response in facilitating C-NHEJ and discuss implications of our findings for potential functions of these factors.
            </br><a style="float: right;" href="https://doi.org/10.1073/pnas.1121458109">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1038/nature09604" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1038/nature09604" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">January 2011</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>ATM damage response and XLF repair factor are functionally redundant in joining DNA breaks</h3>
            Classical non-homologous DNA end-joining (NHEJ) is a major mammalian DNA double-strand-break (DSB) repair pathway. Deficiencies for classical NHEJ factors, such as XRCC4, abrogate lymphocyte development, owing to a strict requirement for classical NHEJ to join V(D)J recombination DSB intermediates. The XRCC4-like factor (XLF; also called NHEJ1) is mutated in certain immunodeficient human patients and has been implicated in classica
            <span class="dots">...</span>
            <span class="more">
            l NHEJ; however, XLF-deficient mice have relatively normal lymphocyte development and their lymphocytes support normal V(D)J recombination. The ataxia telangiectasia-mutated protein (ATM) detects DSBs and activates DSB responses by phosphorylating substrates including histone H2AX. However, ATM deficiency causes only modest V(D)J recombination and lymphocyte developmental defects, and H2AX deficiency does not have a measurable impact on these processes. Here we show that XLF, ATM and H2AX all have fundamental roles in processing and joining DNA ends during V(D)J recombination, but that these roles have been masked by unanticipated functional redundancies. Thus, combined deficiency of ATM and XLF nearly blocks mouse lymphocyte development due to an inability to process and join chromosomal V(D)J recombination DSB intermediates. Combined XLF and ATM deficiency also severely impairs classical NHEJ, but not alternative end-joining, during IgH class switch recombination. Redundant ATM and XLF functions in classical NHEJ are mediated by ATM kinase activity and are not required for extra-chromosomal V(D)J recombination, indicating a role for chromatin-associated ATM substrates. Correspondingly, conditional H2AX inactivation in XLF-deficient pro-B lines leads to V(D)J recombination defects associated with marked degradation of unjoined V(D)J ends, revealing that H2AX has a role in this process.
            </br><a style="float: right;" href="https://doi.org/10.1038/nature09604">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1084/jem.20092449" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1084/jem.20092449" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">February 2010</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Alternative end-joining catalyzes class switch recombination in the absence of both Ku70 and DNA ligase 4</h3>
            The classical nonhomologous end-joining (C-NHEJ) DNA double-strand break (DSB) repair pathway employs the Ku70/80 complex (Ku) for DSB recognition and the XRCC4/DNA ligase 4 (Lig4) complex for ligation. During IgH class switch recombination (CSR) in B lymphocytes, switch (S) region DSBs are joined by C-NHEJ to form junctions either with short microhomologies (MHs; "MH-mediated" joins) or no homologies ("direct" joins). In the absen
            <span class="dots">...</span>
            <span class="more">
            ce of XRCC4 or Lig4, substantial CSR occurs via "alternative" end-joining (A-EJ) that generates largely MH-mediated joins. Because upstream C-NHEJ components remain in XRCC4- or Lig4-deficient B cells, residual CSR might be catalyzed by C-NHEJ using a different ligase. To address this, we have assayed for CSR in B cells deficient for Ku70, Ku80, or both Ku70 and Lig4. Ku70- or Ku80-deficient B cells have reduced, but still substantial, CSR. Strikingly, B cells deficient for both Ku plus Lig4 undergo CSR similarly to Ku-deficient B cells, firmly demonstrating that an A-EJ pathway distinct from C-NHEJ can catalyze CSR end-joining. Ku-deficient or Ku- plus Lig4-deficient B cells are also biased toward MH-mediated CSR joins; but, in contrast to XRCC4- or Lig4-deficient B cells, generate substantial numbers of direct CSR joins. Our findings suggest that more than one form of A-EJ can function in CSR
            </br><a style="float: right;" href="https://doi.org/10.1084/jem.20092449">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1073/pnas.0915067107" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1073/pnas.0915067107" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">February 2010</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Alternative end-joining catalyzes robust IgH locus deletions and translocations in the combined absence of ligase 4 and Ku70</h3>
            Class switch recombination (CSR) in B lymphocytes is initiated by introduction of multiple DNA double-strand breaks (DSBs) into switch (S) regions that flank immunoglobulin heavy chain (IgH) constant region exons. CSR is completed by joining a DSB in the donor S mu to a DSB in a downstream acceptor S region (e.g., S gamma1) by end-joining. In normal cells, many CSR junctions are mediated by classical nonhomologous end-joining (C-NH
            <span class="dots">...</span>
            <span class="more">
            EJ), which employs the Ku70/80 complex for DSB recognition and XRCC4/DNA ligase 4 for ligation. Alternative end-joining (A-EJ) mediates CSR, at reduced levels, in the absence of C-NHEJ, even in combined absence of Ku70 and ligase 4, demonstrating an A-EJ pathway totally distinct from C-NHEJ. Multiple DSBs are introduced into S mu during CSR, with some being rejoined or joined to each other to generate internal switch deletions (ISDs). In addition, S-region DSBs can be joined to other chromosomes to generate translocations, the level of which is increased by absence of a single C-NHEJ component (e.g., XRCC4). We asked whether ISD and S-region translocations occur in the complete absence of C-NHEJ (e.g., in Ku70/ligase 4 double-deficient B cells). We found, unexpectedly, that B-cell activation for CSR generates substantial ISD in both S mu and S gamma1 and that ISD in both is greatly increased by the absence of C-NHEJ. IgH chromosomal translocations to the c-myc oncogene also are augmented in the combined absence of Ku70 and ligase 4. We discuss the implications of these findings for A-EJ in normal and abnormal DSB repair.
            </br><a style="float: right;" href="https://doi.org/10.1073/pnas.0915067107">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1038/nature08159" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1038/nature08159" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">July 2009</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Mechanisms promoting translocations in editing and switching peripheral B cells</h3>
            Variable, diversity and joining gene segment (V(D)J) recombination assembles immunoglobulin heavy or light chain (IgH or IgL) variable region exons in developing bone marrow B cells, whereas class switch recombination (CSR) exchanges IgH constant region exons in peripheral B cells. Both processes use directed DNA double-strand breaks (DSBs) repaired by non-homologous end-joining (NHEJ). Errors in either V(D)J recombination or CSR c
            <span class="dots">...</span>
            <span class="more">
            an initiate chromosomal translocations, including oncogenic IgH locus (Igh) to c-myc (also known as Myc) translocations of peripheral B cell lymphomas. Collaboration between these processes has also been proposed to initiate translocations. However, the occurrence of V(D)J recombination in peripheral B cells is controversial. Here we show that activated NHEJ-deficient splenic B cells accumulate V(D)J-recombination-associated breaks at the lambda IgL locus (Igl), as well as CSR-associated Igh breaks, often in the same cell. Moreover, Igl and Igh breaks are frequently joined to form translocations, a phenomenon associated with specific Igh-Igl co-localization. Igh and c-myc also co-localize in these cells; correspondingly, the introduction of frequent c-myc DSBs robustly promotes Igh-c-myc translocations. Our studies show peripheral B cells that attempt secondary V(D)J recombination, and determine a role for mechanistic factors in promoting recurrent translocations in tumours.
            </br><a style="float: right;" href="https://doi.org/10.1038/nature08159">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="  " class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="  " class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">December 2003</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Inactivation of wild-type p53 protein function by reactive oxygen and nitrogen species in malignant glioma cells</h3>
            Malignant gliomas are the most common primary brain tumors in adults, and the most malignant form, glioblastoma multiforme (GBM), is usually rapidly fatal. Most GBMs do not have p53 mutations, although the p53 tumor suppressor pathway appears to be inactivated. GBMs grow in a hypoxic and inflammatory microenvironment, and increased levels of the free radicals nitric oxide (NO) and superoxide () occur in these malignancies in vivo. 
            <span class="dots">...</span>
            <span class="more">
            Peroxynitrite (ONOO(-)) is a highly reactive molecule produced by excess NO and that can posttranslationally modify and inactivate proteins, especially zinc finger transcription factors such as p53. We demonstrated previously that GBMs have evidence of tyrosine nitration, the "footprint" of peroxynitrite-mediated protein modification in vivo, and that peroxynitrite could inhibit the specific DNA binding ability of wild-type p53 protein in glioma cells in vitro. Here we show that both authentic peroxynitrite and SIN-1 (3-morpholinosydnonimine hydrochloride), a molecule that decomposes into NO and to form peroxynitrite, can inhibit wild-type p53 function in malignant glioma cells. Concentrations of peroxynitrite associated with a tumor inflammatory environment caused dysregulation of wild-type p53 transcriptional activity and downstream p21(WAF1) expression.
            </br><a style="float: right;" href="https://doi.org/  ">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   


<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1093/nar/27.13.2770" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1093/nar/27.13.2770" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">July 1999</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Long 5' leaders inhibit removal of a 3' trailer from a precursor tRNA by mammalian tRNA 3' processing endoribonuclease</h3>
            Mammalian tRNA 3' processing endoribonuclease (3' tRNase) can remove a 3' trailer from various pre-tRNAs without 5' leader nucleotides. To examine how 5[prime] leader sequences affect 3' processing efficiency, we performed in vitro 3' processing reactions with purified pig 3' tRNase and pre-tRNAArgs containing a 13-nt 3' trailer and a 5[prime] leader of various lengths. The 3' processing was slightly stimulated by 5[prime] leaders 
            <span class="dots">...</span>
            <span class="more">
            containing up to 7 nt, whereas leaders of 9 nt or longer severely inhibited the reaction. Structure probing indicated that the 5' leader sequences had little effect on pre-tRNA folding. Similar results were obtained using pre-tRNA(Val)s containing a 5' leader of various lengths. We also investigated whether 3'tRNase can remove 3' trailers that are stably base-paired with 5' leaders to form an extended acceptor stem. Even such small 5' leaders as 3 and 6 nt, when base-paired with a 3' trailer, severely hindered removal of the 3' trailer by 3' tRNase.
            </br><a style="float: right;" href="https://doi.org/10.1093/nar/27.13.2770">View on publisher site</a></span>
            
            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>   
