<!DOCTYPE html>
<html lang="en" dir="ltr">

<head><?php include "./head.php" ?></head>

<body>

<div class="container-header">
    <img src="img/publications2.jpg" id="header-img" alt="Publications Photo" />
    <h1>our publications</h1>
    <?php include "./nvbar.php" ?>
</div>

<!-- main section open -->
<div class='container-body'>
    <div class='cotainer-content'>


<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1126/sciimmunol.abl9464" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1126/sciimmunol.abl9464" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/generic.png">
        <p style="text-align: left;">April 20222</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>SARS-CoV-2 epitope-specific CD4+ memory T cell responses across COVID-19 disease severity and antibody durability</h3>CD4+ T cells are central to long-term immunity against viruses through the functions of T helper 1 (TH1) and T follicular helper (TFH) cell subsets. To better understand the role of these subsets in coronavirus disease 2019 (COVID-19) immunity, we conducted a longitudinal study of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)–specific CD4+ T cell and antibody responses in convalescent individuals who seroconverted during the first wave of the pandemic in Boston, MA, USA, across a range of COVID-19 disease severities. Analyses of spike (S) and nucleocapsid (N) epitope–specific CD4+ T cells using peptide and major histocompatibility complex class II (pMHCII) tetramers<span class="dots">...</span>
            <span class="more">
            demonstrated expanded populations of T cells recognizing the different SARS-CoV-2 epitopes in most individuals compared with prepandemic controls. Individuals who experienced a milder disease course not requiring hospitalization had a greater percentage of circulating TFH (cTFH) and TH1 cells among SARS-CoV-2–specific cells. Analysis of SARS-CoV-2–specific CD4+ T cells responses in a subset of individuals with sustained anti-S antibody responses after viral clearance also revealed an increased proportion of memory cTFH cells. Our findings indicate that efficient early disease control also predicts favorable long-term adaptive immunity.
            </br><a style="float: right;" href="https://doi.org/10.1126/sciimmunol.abl9464">View on publisher site</a></span>
            
            <span class="read">read more</span>
            <!-- <b href="#" class="read">read more...</b> -->
            </div>
        </div>
    </div>
</div>
</div>   

<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1038/s41588-022-01113-z" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1038/s41588-022-01113-z" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/nature_genetics_54_8.png">
        <p style="text-align: left;">July 2022</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Genetic regulation of OAS1 nonsense-mediated decay underlies association with COVID-19 hospitalization in patients of European and African ancestries</h3>
            The chr12q24.13 locus encoding OAS1–OAS3 antiviral proteins has been associated with coronavirus disease 2019 (COVID-19) susceptibility. Here, we report genetic, functional and clinical insights into this locus in relation to COVID-19 severity. In our analysis of patients of European (n = 2,249) and African (n = 835) ancestries with hospitalized versus nonhospitalized COVID-19, the risk of hospitalized disease was associated with a common OAS1 haplotype, which was also associated with reduced severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) clearance in a clinical trial with pegIFN-λ1.<span class="dots">...</span>
            <span class="more">
            Bioinformatic analyses and in vitro studies reveal the functional contribution of two associated OAS1 exonic variants comprising the risk haplotype. Derived human-specific alleles rs10774671-A and rs1131454-A decrease OAS1 protein abundance through allele-specific regulation of splicing and nonsense-mediated decay (NMD). We conclude that decreased OAS1 expression due to a common haplotype contributes to COVID-19 severity. Our results provide insight into molecular mechanisms through which early treatment with interferons could accelerate SARS-CoV-2 clearance and mitigate against severe COVID-19. 
            </br><a style="float: right;" href="https://doi.org/10.1038/s41588-022-01113-z">View on publisher site</a></span>

                <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>

<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1126/sciimmunol.abp8328" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1126/sciimmunol.abp8328" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/science_immunology.png">
        <p style="text-align: left;">May 2022</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Immune recall improves antibody durability and breadth to SARS-CoV-2 variants</h3>
            Key features of immune memory are greater and faster antigen-specific antibody responses to repeat infection. In the setting of immune-evading viral evolution, it is important to understand how far antibody memory recognition stretches across viral variants when memory cells are recalled to action by repeat invasions. It is also important to understand how immune recall influences longevity of secreted antibody responses. We analyzed SARS-CoV-2 variant recognition, dynamics of memory B cells and secreted antibody over time after infection, vaccination, and boosting. We find that a two-dose SARS-CoV-2 vaccination regimen given after natural infection generated greater longitudinal antibody<span class="dots">...</span> 

            <span class="more">
            stability and induced maximal antibody magnitudes with enhanced breadth across Beta, Gamma, Delta and Omicron variants. A homologous 3rd mRNA vaccine dose in COVID-naïve individuals conferred greater cross-variant evenness of neutralization potency with stability that was equal to the hybrid immunity conferred by infection plus vaccination. Within unvaccinated individuals who recovered from COVID, enhanced antibody stability over time was observed within a subgroup of individuals that recovered more quickly from COVID and harbored significantly more memory B cells cross-reactive to endemic coronaviruses early after infection. These cross-reactive clones map to the conserved S2 region of SARS-CoV-2 spike with higher somatic hypermutation levels and greater target affinity. We conclude that SARS-CoV-2 antigen challenge histories in humans influence not only the speed and magnitude of antibody responses, but also functional cross-variant antibody repertoire composition and longevity.
            </br><a style="float: right;" href="https://doi.org/10.1126/sciimmunol.abp8328">View on publisher site</a></span>

            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>

<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1126/sciimmunol.abo3425" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1126/sciimmunol.abo3425" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/science_immunology_7_71_may_2022.jpeg">
        <p style="text-align: left;">May 2022</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Antibodies induced by an ancestral SARS-CoV-2 strain that cross-neutralize variants from Alpha to Omicron BA.1</h3>
            Neutralizing antibodies that recognize the SARS-CoV-2 spike glycoprotein are the principal host defense against viral invasion. Variants of SARS-CoV-2 bear mutations that allow escape from neutralization by many human antibodies, especially those in widely distributed (“public”) classes. Identifying antibodies that neutralize these variants of concern and determining their prevalence are important goals for understanding immune protection. To determine the Delta and Omicron BA.1 variant specificity of B cell repertoires established by an initial Wuhan strain infection, we measured neutralization potencies of 73 antibodies from an unbiased survey of the early memory B cell response. Antibodies recognizing each of three previously defined epitopic regions on the spike receptor binding domain (RBD) varied in neutralization potency and variant-escape resistance.<span class="dots">...</span>
            <span class="more">
            The ACE2 binding surface (“RBD-2”) harbored the binding sites of neutralizing antibodies with the highest potency but with the greatest sensitivity to viral escape; two other epitopic regions on the RBD (“RBD-1” and “RBD-3”) bound antibodies of more modest potency but greater breadth. The structures of several Fab:spike complexes that neutralized all five variants of concern tested, including one Fab each from the RBD-1, -2, and -3 clusters, illustrated the determinants of broad neutralization and showed that B cell repertoires can have specificities that avoid immune escape driven by public antibodies. The structure of the RBD-2 binding, broad neutralizer shows why it retains neutralizing activity for Omicron BA.1, unlike most others in the same public class. Our results correlate with real-world data on vaccine efficacy, which indicate mitigation of disease caused by Omicron BA.1.
            </br><a style="float: right;" href="https://doi.org/10.1126/sciimmunol.abo3425">View on publisher site</a></span>

            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>

<div class="publication">
<div class="flex-container">
    <div style="flex-basis:23%; position:relative;margin-right: 12px">
        <span style="position:absolute; top: 190px; left:40px;" class="dot"></span>
        <div style="position:absolute; top:180px; left:108px;" data-doi="10.1126/sciimmunol.abo3425" class="__dimensions_badge_embed__" data-style="small_circle" data-legend="hover-right"></div>
        <div style="position:absolute; top:180px; left:27px;" data-doi="10.1126/sciimmunol.abo3425" class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" ></div>
        <img style="width:200px; min-width:200px;" src="publications/science_immunology_7_71_may_2022.jpeg">
        <p style="text-align: left;">May 2022</p>
    </div>
    <div style="flex-basis:74%">
        <div class="post">
            <div><h3>Antibodies induced by an ancestral SARS-CoV-2 strain that cross-neutralize variants from Alpha to Omicron BA.1</h3>
            Neutralizing antibodies that recognize the SARS-CoV-2 spike glycoprotein are the principal host defense against viral invasion. Variants of SARS-CoV-2 bear mutations that allow escape from neutralization by many human antibodies, especially those in widely distributed (“public”) classes. Identifying antibodies that neutralize these variants of concern and determining their prevalence are important goals for understanding immune protection. To determine the Delta and Omicron BA.1 variant specificity of B cell repertoires established by an initial Wuhan strain infection, we measured neutralization potencies of 73 antibodies from an unbiased survey of the early memory B cell response. Antibodies recognizing each of three previously defined epitopic regions on the spike receptor binding domain (RBD) varied in neutralization potency and variant-escape resistance.<span class="dots">...</span>
            <span class="more">
            The ACE2 binding surface (“RBD-2”) harbored the binding sites of neutralizing antibodies with the highest potency but with the greatest sensitivity to viral escape; two other epitopic regions on the RBD (“RBD-1” and “RBD-3”) bound antibodies of more modest potency but greater breadth. The structures of several Fab:spike complexes that neutralized all five variants of concern tested, including one Fab each from the RBD-1, -2, and -3 clusters, illustrated the determinants of broad neutralization and showed that B cell repertoires can have specificities that avoid immune escape driven by public antibodies. The structure of the RBD-2 binding, broad neutralizer shows why it retains neutralizing activity for Omicron BA.1, unlike most others in the same public class. Our results correlate with real-world data on vaccine efficacy, which indicate mitigation of disease caused by Omicron BA.1.
            </br><a style="float: right;" href="https://doi.org/10.1126/sciimmunol.abo3425">View on publisher site</a></span>

            <span class="read">read more</span>
            </div>
        </div>
    </div>
</div>
</div>

<br/><br/><br/>
<br/><br/><br/>

<script>

$(document).ready(function(){
	$(".read").click(function(){
		$(this).prev().toggle();
		$(this).siblings('.dots').toggle();
		if($(this).text()=='read more'){
			$(this).text('read less');
		}
		else{
			$(this).text('read more');
		}
	});
});
</script>

</div>
</div>
<!-- main section close -->
<!-- footer open -->
<!-- <P style="padding-top: 15px;" ></P> -->
<?php include "./footer.php" ?>
<!-- footer close -->
</body>  
</html>